modules/tinymce/src/core/main/ts/api/html/Schema.ts

```diff
@@ -464,7 +466,7 @@ const Schema = (settings?: SchemaSettings): Schema => {
   const blockElementsMap = createLookupTable('block_elements', 'hr table tbody thead tfoot ' +
     'th tr td li ol ul caption dl dt dd noscript menu isindex option ' +
     'datalist select optgroup figcaption details summary', textBlockElementsMap);
-  const textInlineElementsMap = createLookupTable('text_inline_elements', 'span strong b em i font strike u var cite ' +
+  const textInlineElementsMap = createLookupTable('text_inline_elements', 'span strong b em i font s strike u var cite ' +
```

Comment:
Does this need a changelog entry as it looks like an additional bug fix?

Reply:
Good point. I added a couple more changelog entries for the other schema fixes I made